src/app/modules/gb-side-panel-module/accordion-components/gb-variables/gb-variables.component.html

```diff
@@ -17,10 +17,10 @@
               (click)="importVariables()"
               type="button">
         <span>import</span>
-        <input [id]="fileElementId"
-               type="file"
-               style="display: none"/>
       </button>
+      <input [id]="variableFileElementId"
```

Comment:
This breaks functionality, because `variableFileElementId` is not defined.

Reply:
I cannot reproduce the twice importing of variables. Let's investigate tomorrow.